Create ui-c8y-1021-80-1-when-using-virtual-scroll-the-list-is-not-always-showing-all.md (#3448)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-80-1-when-using-virtual-scroll-the-list-is-not-always-showing-all.md

---
date: ""

> title: Virtual scroll does not always show all items in a list

Reviewer: ```suggestion
title: Virtual scroll now always shows all items in a list
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63835
version: 1021.80.1
---

> Co-authored-by: Lukasz Janusz <lukasz.janusz@cumuloicty.com>

Reviewer 2: ```suggestion
In places where virtual scroll was implemented, such as `Data points table`, on rare occasions not the entire list would load. This issue has now been resolved.
```

> In places where virtual scroll was implemented, such as `Data points table`, on rare occasions not the entire list would load. This issue has now been resolved.

Reviewer: ```suggestion
In places where virtual scroll was implemented, such as the "Data points table" widget, it could happen on rare occasions that not the entire list was loaded. This issue has now been resolved. Virtual scroll now always shows all list items.
```
